src/components/tests/Tests.jsx

```diff
@@ -10,7 +10,7 @@ function Tests() {
   const getTestsHook = () => {
     const run = async () => {
       try {
-        const testsData = await apiClient.getTestsTemp();
+        const testsData = await apiClient.getTests();
```

Comment:
Wondering if we should sort by `createdAt` before displaying, just to be sure that the order is right?